### PR TITLE
feat: add boehringer ingelheim icon

### PR DESCRIPTION
**Pull request by SimonGolms**
boehringeringelheim

**Issue:** #6033
Alexa rank: 124,924 https://www.alexa.com/siteinfo/boehringer-ingelheim.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- I used the [svg](https://cd.boehringer-ingelheim.com/template/Assets/images/header/bi-logo_web_small.svg) icon in the banner of their corporate design [website](https://cd.boehringer-ingelheim.com/)
- The brand color is`#003366`, which can be found in the registered area. However the color can also be found in the header of the cd website.
- I used Sketch for extracting the logomark and resizing. As this logo is also used as favicon and for the social profile image
- I used SVGOMG with a precision of 5 for further optimization

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1223,6 +1223,11 @@
             "hex": "0066B1",
             "source": "https://www.bmw.de/"
         },
+        {
+            "title": "Boehringer Ingelheim",
+            "hex": "003366",
+            "source": "https://cd.boehringer-ingelheim.com/template/Assets/images/header/bi-logo_web_small.svg"
```
Comment (service-paradis) [inline _data/simple-icons.json:1229]: Can you go with https://cd.boehringer-ingelheim.com for the source? We generally prefer links to pages over files to reduce the possibility of future "link rot".

File: icons/boehringeringelheim.svg [hunk 2]
```diff
@@ -0,0 +1 @@
..." viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Boehringer Ingelheim</title><path d="M5.40571 22.02924c-1.71748-1.2377-2.62725-2.14684-3.486-3.4611C.68202 16.64904 0 14.3242 0 12.02551 0 5.38082 5.3553 0 11.99936 0c6.61917 0 12 5.38083 12 12 0 2.7287-.93466 5.45675-2.57685 7.4779-.68201.85939-1.26259 1.36404-2.8295 2.55134v-7.02366h1.1873v4.80026c.88427-1.03546 1.31363-1.66708 1.743-2.42501.88426-1.56627 1.36467-3.51151 1.36467-5.38147 0-6.06348-4.87618-10.98942-10.88862-10.98942-6.03796 0-10.88798 4.92594-10.88798 11.0647 0 2.80462.98506 5.28067 3.08214 7.73056v-4.79962h1.21219v7.02366zm2.37525 1.21282c-.22713-.0504-.75794-.30304-1.1873-.5308v-7.70568h1.1873v8.23649zM15.00622 6.3155L12 3.99129l-3.00622 2.3242-.6565-.85938L12 2.60173l3.66335 2.85438-.65713.85938zm-4.82514 17.55819c-.42937-.07592-.45489-.07592-.60609-.12696-.12632-.02552-.17672-.02552-.58057-.12633V7.19974h1.1873v16.67394h-.00064zm2.3995.1008C12.37832 24 12.32792 24 12.0504 24c-.32856 0-.45489 0-.65713-.02552V7.19974h1.1873v16.77474zm2.40012-.37897c-.37897.12633-.70753.20225-1.1873.30305V7.19974h1.1873v16.39577zm2.42565-.88425c-.37897.17736-.6061.27816-1.21283.55569v-8.26137h1.21283v7.70568z"/></svg>
```
Comment (service-paradis) [inline icons/boehringeringelheim.svg:1]: I'm seeing some differences when comparing to the original. Ex.
image
image

**Review by service-paradis — CHANGES_REQUESTED** [review]
Than you @SimonGolms for your contribution!
I made some comments about the icon. Can you take a look at it?

File: icons/boehringeringelheim.svg (hunk 2 above)
Comment (SimonGolms) [reply icons/boehringeringelheim.svg]: @service-paradis I updated the icon and used again Sketch to extract the trademark and SVGOMG with a precision of 5 for further optimization. I hope this solves the difference as a precision of 6 will cause an error from linter.

**Review by service-paradis — APPROVED** [review]
Thank you @SimonGolms for your contribution! 💯

**Pull request merged**